sfccdevops/explorer-exclude-vscode-extension
# Added 'explorerExclude.showPicker' configuration option, which when false will NOT show the file picker, but only hide the currently selected item

#### What's this PR do?

It adds a check in util.js for a configuration/setting option: "explorerExclude.showPicker". 
If this is true (or undefined), the extension works as is... if it is false it will NOT show any file picker/choose after clicking 'Add to hidden items...', but will only hide the already selected item. 

#### Where should the reviewer start?

util.js line 383

#### How should this be manually tested?

Add a vscode setting called:  "explorerExclude.showPicker": false and then try to hide any selected item. Set the value to true and confirm the default behavior works.

#### Definition of Done:

- [  ] You have verified that `npm test` passes without issue
